mailer.php

```diff
@@ -1,37 +0,0 @@
-<?php
-require_once './inc/functions.php';
-$conn = mysqli_connect( 'sql6.freemysqlhosting.net' , 'sql6519889' , 'SNTJXSWpSs' , 'sql6519889' );
```

Comment:
Use environment variables, why have you pushed password to GitHub?

Reply:
okay so I can use any variable on github the original code is not req?